tutorials/enricher/pom.xml

```diff
@@ -13,9 +13,9 @@
 
   <properties>
     <!-- change this if you desire to use a different version of Apex Core -->
-    <apex.version>3.4.0</apex.version>
+    <apex.version>3.6.0-dt20161215</apex.version>
```

Comment:
We shouldn't be using special build versions; if the latest release (e.g. 3.5.0) is not adequate we should use 3.6.0-SNAPSHOT